### ISSUE TITLE
conda no longer showing download summary

### ISSUE THREAD

**Issue by asmeurer** [conda/conda#4348]
Here is some output from a conda command. There is no longer a summary of what packages are to be downloaded, and the total download size. It also no longer has untarring and linking progress bars.  The error at the end seems to be unrelated.

```
$conda update --all
Fetching package metadata .................
Solving package specifications: .

Package plan for installation in environment /Users/aaronmeurer/anaconda3:

The following NEW packages will be INSTALLED:

    astroid:                            1.4.9-py35_0                       defaults
    isort:                              4.2.5-py35_0                       conda-forge
    lazy-object-proxy:                  1.2.2-py35_0                       conda-forge
    libuv:                              1.10.1-0                           conda-forge
    pylint:                             1.6.4-py35_0                       conda-forge
    s3transfer:                         0.1.10-py35_0                      defaults
    sortedcontainers:                   1.5.7-py35_0                       defaults
    wrapt:                              1.10.8-py35_0                      conda-forge

The following packages will be UPDATED:

    anaconda-client:                    1.5.1-py35_0                       conda-forge --> 1.6.0-py35_0                       defaults
    arrow:                              0.8.0-py35_0                       conda-forge --> 0.10.0-py35_0                      conda-forge
    astropy:                            1.2.1-np111py35_0                  defaults    --> 1.3-np111py35_0                    conda-forge
    automake:                           1.14-pl5.20.3.1_2                  conda-forge --> 1.15-0                             defaults
    backports.shutil_get_terminal_size: 1.0.0-py35_0                       conda-forge --> 1.0.0-py35_1                       conda-forge
    beautifulsoup4:                     4.5.1-py35_0                       defaults    --> 4.5.3-py35_0                       defaults
    bokeh:                              0.12.2-py35_0                      defaults    --> 0.12.4-py35_0                      defaults
    boost:                              1.61.0-py35_1                      conda-forge --> 1.63.0-py35_2                      conda-forge
    boto:                               2.42.0-py35_0                      defaults    --> 2.45.0-py35_0                      defaults
    boto3:                              1.3.1-py35_0                       defaults    --> 1.4.3-py35_0                       defaults
    botocore:                           1.4.52-py35_0                      defaults    --> 1.4.90-py35_0                      defaults
    bottleneck:                         1.1.0-np111py35_0                  defaults    --> 1.2.0-np111py35_0                  conda-forge
    ca-certificates:                    2016.8.31-0                        conda-forge --> 2016.9.26-0                        conda-forge
    certifi:                            2016.8.31-py35_0                   conda-forge --> 2016.9.26-py35_0                   conda-forge
    cffi:                               1.7.0-py35_0                       defaults    --> 1.9.1-py35_0                       defaults
    click:                              6.6-py35_1                         conda-forge --> 6.7-py35_0                         defaults
    cloog:                              0.18.1-0                           asmeurer    --> 0.18.1-0                           defaults
    cloudpickle:                        0.2.1-py35_2                       conda-forge --> 0.2.2-py35_2                       conda-forge
    cmake:                              3.6.1-0                            conda-forge --> 3.7.1-4                            conda-forge
    conda-build:                        2.1.1-py35_0                       defaults    --> 2.1.1-py35_0                       conda-forge
    conda-build-all:                    1.0.0-py35_0                       conda-forge --> 1.0.1-py35_1                       conda-forge
    contextlib2:                        0.5.3-py35_0                       conda-forge --> 0.5.4-py35_0                       conda-forge
    coverage:                           4.2-py35_0                         conda-forge --> 4.3.1-py35_0                       conda-forge
    cryptography:                       1.5-py35_0                         defaults    --> 1.7.1-py35_0                       defaults
    curl:                               7.49.1-1                           conda-forge --> 7.49.1-2                           conda-forge
    cython:                             0.24.1-py35_0                      conda-forge --> 0.25.2-py35_0                      conda-forge
    cytoolz:                            0.8.0-py35_0                       conda-forge --> 0.8.2-py35_0                       conda-forge
    dask:                               0.11.0-py35_0                      defaults    --> 0.13.0-py35_0                      conda-forge
    datashader:                         0.3.0-py35_0                       defaults    --> 0.4.0-py35_0                       defaults
    datashape:                          0.5.2-py35_0                       defaults    --> 0.5.4-py35_0                       defaults
    dbus:                               1.10.10-0                          conda-forge --> 1.10.10-1                          conda-forge
    decorator:                          4.0.10-py35_0                      conda-forge --> 4.0.11-py35_0                      defaults
    dill:                               0.2.5-py35_0                       conda-forge --> 0.2.5-py35_1                       conda-forge
    distributed:                        1.13.2-py35_0                      conda-forge --> 1.15.1-py35_0                      conda-forge
    doctr:                              1.3.2-py35_0                       conda-forge --> 1.4.1-py35_0                       conda-forge
    docutils:                           0.12-py35_0                        conda-forge --> 0.13.1-py35_0                      defaults
    ffmpeg:                             2.8.6-2                            conda-forge --> 2.8.6-3                            conda-forge
    filelock:                           2.0.6-py35_0                       conda-forge --> 2.0.7-py35_0                       defaults
    future:                             0.15.2-py35_0                      conda-forge --> 0.16.0-py35_1                      defaults
    gevent:                             1.1.2-py35_0                       defaults    --> 1.2.1-py35_0                       defaults
    gitdb:                              0.6.4-py35_0                       conda-forge --> 0.6.4-py35_1                       conda-forge
    gitpython:                          2.0.8-py35_0                       conda-forge --> 2.1.1-py35_0                       conda-forge
    gmpy2:                              2.0.7-py35_1                       conda-forge --> 2.0.8-py35_0                       conda-forge
    greenlet:                           0.4.10-py35_0                      defaults    --> 0.4.11-py35_0                      defaults
    h5py:                               2.6.0-np111py35_6                  conda-forge --> 2.6.0-np111py35_7                  conda-forge
    hdf5:                               1.8.17-3                           conda-forge --> 1.8.17-9                           conda-forge
    idna:                               2.1-py35_0                         conda-forge --> 2.2-py35_0                         defaults
    ipykernel:                          4.5.0-py35_0                       conda-forge --> 4.5.2-py35_0                       conda-forge
    ipywidgets:                         5.2.2-py35_0                       conda-forge --> 5.2.2-py35_2                       conda-forge
    jdcal:                              1.2-py35_1                         defaults    --> 1.3-py35_0                         defaults
    jinja2:                             2.8-py35_1                         conda-forge --> 2.9.4-py35_0                       defaults
    jupyter_core:                       4.2.0-py35_0                       conda-forge --> 4.2.1-py35_0                       conda-forge
    jupyterlab:                         0.2.0-py35_1                       conda-forge --> 0.12.1-py35_0                      conda-forge
    libevent:                           2.0.21-0                           defaults    --> 2.0.22-0                           conda-forge
    libiconv:                           1.14-3                             conda-forge --> 1.14-4                             conda-forge
    libnetcdf:                          4.4.1-0                            conda-forge --> 4.4.1.1-2                          conda-forge
    libpng:                             1.6.24-0                           conda-forge --> 1.6.28-0                           conda-forge
    libtiff:                            4.0.6-6                            conda-forge --> 4.0.6-7                            conda-forge
    libxml2:                            2.9.4-1                            conda-forge --> 2.9.4-3                            conda-forge
    libxslt:                            1.1.29-1                           conda-forge --> 1.1.29-2                           conda-forge
    livereload:                         2.4.1-py35_0                       conda-forge --> 2.5.1-py35_0                       conda-forge
    llvmlite:                           0.13.0-py35_0                      defaults    --> 0.15.0-py35_0                      defaults
    locket:                             0.2.0-py35_0                       conda-forge --> 0.2.0-py35_1                       conda-forge
    lxml:                               3.6.4-py35_0                       conda-forge --> 3.7.2-py35_0                       conda-forge
    m4:                                 1.4.17-0                           conda-forge --> 1.4.17-1                           conda-forge
    mako:                               1.0.4-py35_0                       conda-forge --> 1.0.6-py35_0                       defaults
    markupsafe:                         0.23-py35_0                        conda-forge --> 0.23-py35_1                        conda-forge
    mccabe:                             0.5.0-py35_0                       conda-forge --> 0.5.3-py35_0                       conda-forge
    mkl:                                11.3.3-0                           defaults    --> 2017.0.1-0                         defaults
    mpc:                                1.0.3-1                            conda-forge --> 1.0.3-3                            conda-forge
    mpfr:                               3.1.4-1                            conda-forge --> 3.1.5-0                            conda-forge
    mpmath:                             0.19-py35_1                        conda-forge --> 0.19-py35_2                        conda-forge
    msgpack-python:                     0.4.7-py35_0                       defaults    --> 0.4.8-py35_0                       conda-forge
    multipledispatch:                   0.4.8-py35_0                       conda-forge --> 0.4.9-py35_0                       defaults
    nbformat:                           4.1.0-py35_0                       conda-forge --> 4.2.0-py35_0                       conda-forge
    ncurses:                            5.9-9                              conda-forge --> 5.9-10                             conda-forge
    netcdf4:                            1.2.4-np111py35_2                  conda-forge --> 1.2.7-np111py35_0                  conda-forge
    networkx:                           1.11-py35_0                        defaults    --> 1.11-py35_0                        conda-forge
    nltk:                               3.2.1-py35_0                       conda-forge --> 3.2.2-py35_0                       defaults
    nodejs:                             4.5.0-0                            conda-forge --> 6.6.0-0                            conda-forge
    notebook:                           4.2.3-py35_0                       conda-forge --> 4.3.1-py35_0                       conda-forge
    numba:                              0.28.1-np111py35_0                 defaults    --> 0.30.1-np111py35_0                 defaults
    numexpr:                            2.6.1-np111py35_0                  conda-forge --> 2.6.1-np111py35_1                  conda-forge
    numpy:                              1.11.1-py35_blas_openblas_201      conda-forge [blas_openblas] --> 1.11.3-py35_blas_openblas_200      conda-forge [blas_openblas]
    numpydoc:                           0.6.0-py35_0                       defaults    --> 0.6.0-py35_0                       conda-forge
    openblas:                           0.2.18-5                           conda-forge --> 0.2.19-0                           conda-forge
    openpyxl:                           2.3.2-py35_0                       defaults    --> 2.4.1-py35_0                       defaults
    openssl:                            1.0.2h-2                           conda-forge --> 1.0.2j-0                           defaults
    pandas:                             0.18.1-np111py35_1                 conda-forge --> 0.19.2-np111py35_1                 conda-forge
    partd:                              0.3.6-py35_0                       conda-forge --> 0.3.7-py35_0                       conda-forge
    path.py:                            8.2.1-py35_0                       conda-forge --> 10.0-py35_0                        defaults
    pathlib2:                           2.1.0-py35_0                       conda-forge --> 2.2.0-py35_0                       defaults
    perl:                               5.20.3.1-1                         conda-forge --> 5.22.2.1-0                         conda-forge
    pillow:                             3.3.1-py35_0                       conda-forge --> 4.0.0-py35_0                       conda-forge
    pip:                                8.1.2-py35_0                       conda-forge --> 9.0.1-py35_0                       conda-forge
    pkg-config:                         0.28-1                             conda-forge --> 0.29.1-1                           conda-forge
    pkginfo:                            1.3.2-py35_0                       defaults    --> 1.4.1-py35_0                       defaults
    ply:                                3.9-py35_0                         defaults    --> 3.9-py35_0                         conda-forge
    prompt_toolkit:                     1.0.7-py35_0                       conda-forge --> 1.0.9-py35_0                       conda-forge
    psutil:                             4.3.1-py35_0                       defaults    --> 5.0.1-py35_0                       defaults
    pycodestyle:                        2.0.0-py35_0                       conda-forge --> 2.2.0-py35_0                       conda-forge
    pycparser:                          2.14-py35_0                        conda-forge --> 2.17-py35_0                        conda-forge
    pygithub:                           1.27.1-py35_0                      conda-forge --> 1.29-py35_0                        conda-forge
    pyparsing:                          2.1.8-py35_0                       conda-forge --> 2.1.10-py35_0                      conda-forge
    pyqt:                               4.11.4-py35_0                      conda-forge --> 4.11.4-py35_2                      conda-forge
    pytest-cov:                         2.3.1-py35_0                       conda-forge --> 2.4.0-py35_0                       conda-forge
    python-dateutil:                    2.5.3-py35_0                       conda-forge --> 2.6.0-py35_0                       conda-forge
    python-symengine:                   0.1.0.dev0-py35_0                  conda-forge --> 0.2.0-py35_1                       conda-forge
    pytools:                            2016.2.1-py35_0                    conda-forge --> 2016.2.6-py35_0                    conda-forge
    pytz:                               2016.6.1-py35_0                    conda-forge --> 2016.10-py35_0                     conda-forge
    pyzmq:                              15.4.0-py35_0                      conda-forge --> 16.0.2-py35_0                      conda-forge
    qtawesome:                          0.3.3-py35_0                       defaults    --> 0.4.1-py35_0                       defaults
    qtconsole:                          4.2.1-py35_0                       conda-forge --> 4.2.1-py35_1                       conda-forge
    qtpy:                               1.1.1-py35_0                       conda-forge --> 1.1.2-py35_0                       defaults
    ruamel.yaml:                        0.11.11-py35_1                     conda-forge --> 0.13.7-py35_0                      conda-forge
    ruamel_yaml:                        0.11.14-py35_0                     defaults    --> 0.11.14-py35_0                     conda-forge
    s3fs:                               0.0.7-py35_0                       conda-forge --> 0.0.8-py35_0                       conda-forge
    scikit-learn:                       0.17.1-np111py35_blas_openblas_202 conda-forge [blas_openblas] --> 0.18.1-np111py35_1                 defaults
    scipy:                              0.18.0-np111py35_blas_openblas_202 conda-forge [blas_openblas] --> 0.18.1-np111py35_blas_openblas_201 conda-forge [blas_openblas]
    setuptools:                         27.2.0-py35_0                      defaults    --> 32.3.1-py35_0                      conda-forge
    sip:                                4.18-py35_0                        conda-forge --> 4.18-py35_1                        conda-forge
    six:                                1.10.0-py35_0                      conda-forge --> 1.10.0-py35_1                      conda-forge
    smmap:                              0.9.0-py35_0                       conda-forge --> 2.0.1-py35_0                       conda-forge
    sphinxjp.themecore:                 0.2.0-py35_0                       conda-forge --> 0.2.0-py35_1                       conda-forge
    spyder:                             2.3.9-py35_0                       conda-forge --> 3.0.2-py35_0                       defaults
    sqlalchemy:                         1.0.13-py35_0                      defaults    --> 1.1.4-py35_0                       conda-forge
    symengine:                          0.1.0.dev0-0                       conda-forge --> 0.2.0-1                            conda-forge
    sympy:                              1.0-py35_0                         defaults    --> 1.0-py35_0                         conda-forge
    tk:                                 8.5.19-0                           conda-forge --> 8.5.19-1                           conda-forge
    toolz:                              0.8.0-py35_0                       conda-forge --> 0.8.2-py35_0                       conda-forge
    tornado:                            4.4.1-py35_1                       conda-forge --> 4.4.2-py35_0                       conda-forge
    traitlets:                          4.3.0-py35_0                       conda-forge --> 4.3.1-py35_0                       conda-forge
    uritemplate.py:                     0.3.0-py35_0                       conda-forge --> 3.0.2-py35_0                       conda-forge
    versioneer:                         0.16-py35_0                        conda-forge --> 0.17-py35_0                        conda-forge
    virtualenv:                         13.0.1-py35_0                      defaults    --> 15.0.3-py35_0                      defaults
    werkzeug:                           0.11.11-py35_0                     defaults    --> 0.11.15-py35_0                     defaults
    xlsxwriter:                         0.9.3-py35_0                       defaults    --> 0.9.6-py35_0                       defaults
    xlwings:                            0.9.3-py35_0                       defaults    --> 0.10.2-py35_0                      defaults
    xlwt:                               1.1.2-py35_0                       defaults    --> 1.2.0-py35_0                       defaults
    zict:                               0.0.3-py35_0                       conda-forge --> 0.1.1-py35_0                       conda-forge

The following packages will be SUPERCEDED by a higher-priority channel:

    jedi:                               0.9.0-py35_1                       defaults    --> 0.9.0-py35_0                       conda-forge

The following packages will be DOWNGRADED due to dependency conflicts:

    autoconf:                           2.69-pl5.20.3.1_3                  conda-forge --> 2.69-2                             conda-forge
    conda:                              4.3.4-py35_0                       defaults    --> 4.2.15-py35_0                      defaults

ca-certificate 100% |##############################################################################################################| Time: 0:00:00   1.25 MB/s
libuv-1.10.1-0 100% |##############################################################################################################| Time: 0:00:00   3.01 MB/s
m4-1.4.17-1.ta 100% |##############################################################################################################| Time: 0:00:00   5.01 MB/s
nodejs-6.6.0-0 100% |##############################################################################################################| Time: 0:00:02   4.58 MB/s
perl-5.22.2.1- 100% |##############################################################################################################| Time: 0:00:03   4.42 MB/s
autoconf-2.69- 100% |##############################################################################################################| Time: 0:00:00   4.85 MB/s
curl-7.49.1-2. 100% |##############################################################################################################| Time: 0:00:00   4.80 MB/s
hdf5-1.8.17-9. 100% |##############################################################################################################| Time: 0:00:00   4.66 MB/s
libevent-2.0.2 100% |##############################################################################################################| Time: 0:00:00   4.77 MB/s
mpfr-3.1.5-0.t 100% |##############################################################################################################| Time: 0:00:00   4.80 MB/s
openblas-0.2.1 100% |##############################################################################################################| Time: 0:00:03   4.47 MB/s
pkg-config-0.2 100% |##############################################################################################################| Time: 0:00:00   2.28 MB/s
automake-1.15- 100% |##############################################################################################################| Time: 0:00:00   3.39 MB/s
cloog-0.18.1-0 100% |##############################################################################################################| Time: 0:00:00   4.38 MB/s
cmake-3.7.1-4. 100% |##############################################################################################################| Time: 0:00:03   2.74 MB/s
ffmpeg-2.8.6-3 100% |##############################################################################################################| Time: 0:00:10   4.62 MB/s
libnetcdf-4.4. 100% |##############################################################################################################| Time: 0:00:00   4.69 MB/s
libxslt-1.1.29 100% |##############################################################################################################| Time: 0:00:00   4.78 MB/s
mpc-1.0.3-3.ta 100% |##############################################################################################################| Time: 0:00:00   5.71 MB/s
beautifulsoup4 100% |##############################################################################################################| Time: 0:00:00   4.85 MB/s
boost-1.63.0-p 100% |##############################################################################################################| Time: 0:00:03   4.78 MB/s
boto-2.45.0-py 100% |##############################################################################################################| Time: 0:00:00   4.59 MB/s
click-6.7-py35 100% |##############################################################################################################| Time: 0:00:00   6.63 MB/s
cloudpickle-0. 100% |##############################################################################################################| Time: 0:00:00   1.78 MB/s
contextlib2-0. 100% |##############################################################################################################| Time: 0:00:00   3.31 MB/s
coverage-4.3.1 100% |##############################################################################################################| Time: 0:00:00   4.49 MB/s
cython-0.25.2- 100% |##############################################################################################################| Time: 0:00:00   3.94 MB/s
dill-0.2.5-py3 100% |##############################################################################################################| Time: 0:00:00   4.31 MB/s
docutils-0.13. 100% |##############################################################################################################| Time: 0:00:00   4.59 MB/s
filelock-2.0.7 100% |##############################################################################################################| Time: 0:00:00  11.35 MB/s
future-0.16.0- 100% |##############################################################################################################| Time: 0:00:00   4.78 MB/s
gmpy2-2.0.8-py 100% |##############################################################################################################| Time: 0:00:00   1.46 MB/s
greenlet-0.4.1 100% |##############################################################################################################| Time: 0:00:00  25.58 MB/s
idna-2.2-py35_ 100% |##############################################################################################################| Time: 0:00:00   5.91 MB/s
isort-4.2.5-py 100% |##############################################################################################################| Time: 0:00:00   1.45 MB/s
jdcal-1.3-py35 100% |##############################################################################################################| Time: 0:00:00  10.38 MB/s
jedi-0.9.0-py3 100% |##############################################################################################################| Time: 0:00:00   1.19 MB/s
lazy-object-pr 100% |##############################################################################################################| Time: 0:00:00   3.10 MB/s
llvmlite-0.15. 100% |##############################################################################################################| Time: 0:00:01   4.46 MB/s
locket-0.2.0-p 100% |##############################################################################################################| Time: 0:00:00   5.12 MB/s
lxml-3.7.2-py3 100% |##############################################################################################################| Time: 0:00:00   2.26 MB/s
mccabe-0.5.3-p 100% |##############################################################################################################| Time: 0:00:00   9.51 MB/s
msgpack-python 100% |##############################################################################################################| Time: 0:00:00   2.90 MB/s
multipledispat 100% |##############################################################################################################| Time: 0:00:00   8.55 MB/s
numpy-1.11.3-p 100% |##############################################################################################################| Time: 0:00:01   3.32 MB/s
pillow-4.0.0-p 100% |##############################################################################################################| Time: 0:00:00   3.31 MB/s
pkginfo-1.4.1- 100% |##############################################################################################################| Time: 0:00:00   6.96 MB/s
psutil-5.0.1-p 100% |##############################################################################################################| Time: 0:00:00   4.91 MB/s
pycodestyle-2. 100% |##############################################################################################################| Time: 0:00:00   2.68 MB/s
pycparser-2.17 100% |##############################################################################################################| Time: 0:00:00   3.78 MB/s
pygithub-1.29- 100% |##############################################################################################################| Time: 0:00:00   4.04 MB/s
qtpy-1.1.2-py3 100% |##############################################################################################################| Time: 0:00:00  15.83 MB/s
ruamel_yaml-0. 100% |##############################################################################################################| Time: 0:00:00   4.40 MB/s
smmap-2.0.1-py 100% |##############################################################################################################| Time: 0:00:00   9.12 MB/s
sortedcontaine 100% |##############################################################################################################| Time: 0:00:00   5.50 MB/s
sqlalchemy-1.1 100% |##############################################################################################################| Time: 0:00:00   4.44 MB/s
symengine-0.2. 100% |##############################################################################################################| Time: 0:00:00   4.50 MB/s
toolz-0.8.2-py 100% |##############################################################################################################| Time: 0:00:00   4.74 MB/s
uritemplate.py 100% |##############################################################################################################| Time: 0:00:00  13.07 MB/s
versioneer-0.1 100% |##############################################################################################################| Time: 0:00:00   3.07 MB/s
virtualenv-15. 100% |##############################################################################################################| Time: 0:00:00   4.40 MB/s
werkzeug-0.11. 100% |##############################################################################################################| Time: 0:00:00   4.93 MB/s
wrapt-1.10.8-p 100% |##############################################################################################################| Time: 0:00:00 593.46 kB/s
xlsxwriter-0.9 100% |##############################################################################################################| Time: 0:00:00   5.27 MB/s
xlwt-1.2.0-py3 100% |##############################################################################################################| Time: 0:00:00   5.39 MB/s
astroid-1.4.9- 100% |##############################################################################################################| Time: 0:00:00   5.03 MB/s
astropy-1.3-np 100% |##############################################################################################################| Time: 0:00:02   3.62 MB/s
bottleneck-1.2 100% |##############################################################################################################| Time: 0:00:00   4.59 MB/s
cffi-1.9.1-py3 100% |##############################################################################################################| Time: 0:00:00  40.08 MB/s
cytoolz-0.8.2- 100% |##############################################################################################################| Time: 0:00:00   1.12 MB/s
gevent-1.2.1-p 100% |##############################################################################################################| Time: 0:00:00   4.69 MB/s
gitdb-0.6.4-py 100% |##############################################################################################################| Time: 0:00:00   1.34 MB/s
h5py-2.6.0-np1 100% |##############################################################################################################| Time: 0:00:00   3.18 MB/s
mako-1.0.6-py3 100% |##############################################################################################################| Time: 0:00:00   3.63 MB/s
networkx-1.11- 100% |##############################################################################################################| Time: 0:00:00   3.67 MB/s
nltk-3.2.2-py3 100% |##############################################################################################################| Time: 0:00:00   4.59 MB/s
numba-0.30.1-n 100% |##############################################################################################################| Time: 0:00:00   4.60 MB/s
numexpr-2.6.1- 100% |##############################################################################################################| Time: 0:00:00   1.90 MB/s
openpyxl-2.4.1 100% |##############################################################################################################| Time: 0:00:00   4.91 MB/s
partd-0.3.7-py 100% |##############################################################################################################| Time: 0:00:00  18.89 MB/s
pathlib2-2.2.0 100% |##############################################################################################################| Time: 0:00:00  19.65 MB/s
python-symengi 100% |##############################################################################################################| Time: 0:00:00   2.60 MB/s
pytools-2016.2 100% |##############################################################################################################| Time: 0:00:00   4.25 MB/s
qtawesome-0.4. 100% |##############################################################################################################| Time: 0:00:00   6.36 MB/s
scipy-0.18.1-n 100% |##############################################################################################################| Time: 0:00:03   4.30 MB/s
sympy-1.0-py35 100% |##############################################################################################################| Time: 0:00:01   4.62 MB/s
xlwings-0.10.2 100% |##############################################################################################################| Time: 0:00:00   4.71 MB/s
zict-0.1.1-py3 100% |##############################################################################################################| Time: 0:00:00   5.32 MB/s
arrow-0.10.0-p 100% |##############################################################################################################| Time: 0:00:00   5.17 MB/s
botocore-1.4.9 100% |##############################################################################################################| Time: 0:00:00   4.58 MB/s
cryptography-1 100% |##############################################################################################################| Time: 0:00:00   4.82 MB/s
datashape-0.5. 100% |##############################################################################################################| Time: 0:00:00   5.86 MB/s
gitpython-2.1. 100% |##############################################################################################################| Time: 0:00:00   2.03 MB/s
livereload-2.5 100% |##############################################################################################################| Time: 0:00:00   3.52 MB/s
netcdf4-1.2.7- 100% |##############################################################################################################| Time: 0:00:00   4.58 MB/s
pandas-0.19.2- 100% |##############################################################################################################| Time: 0:00:02   4.62 MB/s
pylint-1.6.4-p 100% |##############################################################################################################| Time: 0:00:00   3.76 MB/s
ruamel.yaml-0. 100% |##############################################################################################################| Time: 0:00:00   5.22 MB/s
scikit-learn-0 100% |##############################################################################################################| Time: 0:00:01   4.54 MB/s
anaconda-clien 100% |##############################################################################################################| Time: 0:00:00   6.17 MB/s
bokeh-0.12.4-p 100% |##############################################################################################################| Time: 0:00:01   4.25 MB/s
doctr-1.4.1-py 100% |##############################################################################################################| Time: 0:00:00 822.02 kB/s
pytest-cov-2.4 100% |##############################################################################################################| Time: 0:00:00   1.20 MB/s
s3transfer-0.1 100% |##############################################################################################################| Time: 0:00:00   4.35 MB/s
boto3-1.4.3-py 100% |##############################################################################################################| Time: 0:00:00   3.45 MB/s
conda-4.2.15-p 100% |##############################################################################################################| Time: 0:00:00   2.86 MB/s
dask-0.13.0-py 100% |##############################################################################################################| Time: 0:00:00   2.29 MB/s
numpydoc-0.6.0 100% |##############################################################################################################| Time: 0:00:00   9.14 MB/s
sphinxjp.theme 100% |##############################################################################################################| Time: 0:00:00   7.10 MB/s
conda-build-2. 100% |##############################################################################################################| Time: 0:00:00   3.28 MB/s
s3fs-0.0.8-py3 100% |##############################################################################################################| Time: 0:00:00   4.01 MB/s
conda-build-al 100% |##############################################################################################################| Time: 0:00:00   2.60 MB/s
datashader-0.4 100% |##############################################################################################################| Time: 0:00:00   3.39 MB/s
distributed-1. 100% |##############################################################################################################| Time: 0:00:00   2.39 MB/s
jupyterlab-0.1 100% |##############################################################################################################| Time: 0:00:00   4.12 MB/s
spyder-3.0.2-p 100% |##############################################################################################################| Time: 0:00:01   3.08 MB/s


CondaError: No package 'conda-forge::toolz-0.8.2-py35_0' found in cache directories.


```

**Comment by groutr** [comment]
I'm missing this output too.  Conda 4.3.7.

**Comment by navneethc** [comment]
+1 Exactly what I came here for. As it is, whenever I install or remove packages or environment, I can't be entirely sure if something is happening in the background or if the process is hanging. Eventually the rather unhelpful line `1 file(s) copied.` appears (for installation) after tens of packages are installed and I'm returned to the prompt.

Please bring back the summaries and the progress bars.

Windows 7 | conda 4.3.8

**Comment by nehaljwani** [comment]
IIUC, the change in behaviour was introduced by commit: 24ad47096fe79e9168839f603348b0fdf4abb33b

**Comment by kalefranz** [comment]
This is already fixed in 4.4.
